src/App.js

```diff
@@ -17,16 +27,52 @@ function App() {
     previous: "50 kg x 5",
     kg: "50",
     reps: "9",
-    hola: "bye",
   };
 
+  const listOfButtons = {
+    set: "ADD SET",
+    exercise: "ADD EXERCISE",
+    cancel: "CANCEL WORKOUT"
+  }
+
+  const exercises = {
+    A: "Squat (Barbell)"
+  }
+
+  const workoutNames = {
+    A: "Evening workout"
+  }
+  
+  const Cronometer = () => {
```

Comment:
Why have this function if there's only one option and it will always return the same thing?

Reply:
Because I wanted to introduce a real chronometer, but then I remember that I had to focus on the UI. Saludos!

Reply:
you can keep it then haha. Usually, we want to keep things that are not fully developed outside of the main, but its just us so not a big deal.